Celeste.Mod.mm/MonoModRules.cs

```diff
@@ -1960,6 +1961,15 @@ bgt.un.s	ldloca.s
             }
             if (matches != 4)
                 throw new Exception("Incorrect number of matches for string starting with \"areaselect/card\".");
+
+            cursor.Index = 0;
+
+            // Resize the title if it does not fit.
+            while (cursor.TryGotoNext(instr => instr.MatchLdcR4(-60))) {
+                cursor.Emit(OpCodes.Ldarg_0);
+                cursor.Index++;
+                cursor.Emit(OpCodes.Call, m_FixTitleLength);
+            }
```

Comment:
Everest patches should fail hard wherever possible, similar to the code above this.